src/js/index.js

```diff
@@ -0,0 +1,33 @@
+let balance = 0;
+const operations = [];
+const OperationsType = {
+    EXPENSE: 'EXPENSE',
+    INCOME: 'INCOME',
+}
+/**
+ * {
+ *   type: 'EXPENSE',
+ *   amount: 120,
+ *   description: 'Bill payment'
+ * }
+ * @param {*} operation 
+ */
+function addOperation(operation) {
+    const operationToAdd = operation;
+    operationToAdd.id = new Date().getTime();
+    if(operationToAdd.type === OperationsType.EXPENSE) {
+        balance = balance - operationToAdd.amount;
+    } else if(operationToAdd.type === OperationsType.INCOME) {
+        balance = balance + operationToAdd.amount;
+    }
+    operations.push(operationToAdd);
+}
+function removeOperation() {
+    operations.slice()
+}
+// Find operation
+// Get balance
+// Get operations (lista operazioni)
+function NewBalance(){

```

Comment:
NewBalance should be written in camelCase. So, write it as newBalance.
By the way, the name should be getBalance.